Getting-Started/Setup/Upgrading/upgrading-to-v8.md

```diff
@@ -0,0 +1,80 @@
+---
+versionFrom: 8.1.0
+---
+# Migrating content to Umbraco 8
+
+Umbraco 8 contains a lot of breaking changes and a lot of code has been cleaned up compared to Umbraco 7. Due to this, it will not be possible to do a simple upgrade from Umbraco 7 to Umbraco 8. Instead you will need to **migrate your content** from your Umbraco 7 site into your Umbraco 8 site and then recreate the rest in the new version.
+
+A content migration tool has been implemented in Umbraco 8.1.0, to help you with the transition.
+
+In this guide you can read more about the tool, it's limitations and how to use it in practice.
+
+## Limitations
+
+### Versions supported
+
+The content migration tool is a database migration, which is made for the database schema of Umbraco 7.14+. This means that in order to do the migration the first step is to ensure your Umbraco 7 site is running at least Umbraco 7.14.
+
+### Database types supported
+
+As Umbraco 8 doesn't support MySQL databases the migration will not work when moving from an Umbraco 7 site using MySQL to Umbraco 8 on SQL server for example.
+The database types that are supported are SQL Server and SQL CE.
+
+### Third party property editors
+
+The migration will transform the data stored in third party editors as well, however it will simply be stored as it was in Umbraco 7. If the structure has changed for Umbraco 8 or the property editor doesn't exist you will still be able to find the data in the database, but you would not see it in the backoffice.
+
```

Comment:
You will need to find an equivalent V8 compatible property editor, and migrate the old data format into that of the new property editor's format...

Reply:
new article now on migrating property editors 😁

Reply:
still have my old V6->V7 property editor data migrator console app somewhere...